app/llm-poc-variant-01/docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10-bookworm
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+ENTRYPOINT ["/bin/bash"]
```

Comment:
there's also the option of including the python files directly in the container which means you don't need to run the python command on start up and go directly to the cli. But up to you. (depends quite a lot on whether you expect the end user to make changes to the python files really)

Reply:
Actually, for this instance I would like the container to be an executor/virtual shell around the code, and mount the code into the container. 

The reason for the container is to also isolate the whole environment for execution purposes while the data was read and written to the localmachine

Reply:
This will come to prove useful when I start working on the Cloud implementations